src/components/core/PivotTable.tsx

```diff
@@ -51,6 +81,28 @@ export interface IGridParams {
     api: IGridApi;
 }
 
+export interface IGridCell {
```

Comment:
Add this to interfaces

Reply:
by the way https://github.com/ag-grid/ag-grid/blob/master/packages/ag-grid/src/ts/rendering/cellRenderers/iCellRenderer.ts

Reply:
done